app/models/requests/IdentifierRequest.scala

```diff
@@ -25,7 +25,7 @@ case class IdentifierRequest[A] (
                                   request: Request[A],
                                   credentials: Credentials,
                                   vrn: Vrn,
-                                  iossNumber: String,
+                                  iossNumber: String, //this is optional in registration
```

Comment:
Will remove but should we make this optional to be safe? Unless there is a contract I can look at?